attributes/network/9_account_features/cddl/1_account_multisig.cddl

```diff
@@ -101,19 +101,19 @@ account-multisig-transaction-expire = {
 ; TODO: find a better place for these to live. We cannot just keep updating this file everytime
 ; we have a new transaction type.
 account-multisig-transaction /=
-       { 0 => ledger-transaction-type-send, 1 => ledger.send@param }
-    /  { 0 => account-transaction-type-create, 1 => account.create@param }
-    /  { 0 => account-transaction-type-set-description, 1 => account.setDescription@param }
-    /  { 0 => account-transaction-type-add-roles, 1 => account.addRoles@param }
-    /  { 0 => account-transaction-type-remove-roles, 1 => account.removeRoles@param }
-    /  { 0 => account-transaction-type-delete, 1 => account.delete@param }
-    /  { 0 => account-transaction-type-add-features, 1 => account.addFeatures@param }
-    /  { 0 => account-transaction-type-multisig-submit, 1 => account.multisigSubmitTransaction@param }
-    /  { 0 => account-transaction-type-multisig-approve, 1 => account.multisigApprove@param }
-    /  { 0 => account-transaction-type-multisig-revoke, 1 => account.multisigRevoke@param }
-    /  { 0 => account-transaction-type-multisig-execute, 1 => account.multisigExecute@param }
-    /  { 0 => account-transaction-type-multisig-withdraw, 1 => account.multisigWithdraw@param }
-    /  { 0 => account-transaction-type-multisig-set-defaults, 1 => account.multisigSetDefaults@param }
+       { 0 => event-type-send, 1 => ledger.send@param }
+    /  { 0 => account-event-type-create, 1 => account.create@param }
+    /  { 0 => account-event-type-set-description, 1 => account.setDescription@param }
+    /  { 0 => account-event-type-add-roles, 1 => account.addRoles@param }
+    /  { 0 => account-event-type-remove-roles, 1 => account.removeRoles@param }
+    /  { 0 => account-event-type-delete, 1 => account.delete@param }
```

Comment:
You probably want to do it in another PR, but we need to update the spec to change `delete` with `disable` before merging https://github.com/liftedinit/many-framework/pull/140